components/brave_vpn/mojom/brave_vpn.mojom

```diff
@@ -90,6 +90,7 @@ enum ConnectionState {
   DISCONNECTING,
   CONNECT_NOT_ALLOWED,
   CONNECT_FAILED,
+  UNKNOWN,
```

Comment:
I think you have a rebase problem here. This was removed

Reply:
no, I added it here to get notification on launch about vpn state when it is disconnected. For the case when we disconnected VPN from OS when the browser is closed.

Reply:
where is it used? I don't see anything that ever checks it

Reply:
I agree with @bsclifton https://github.com/brave/brave-core/pull/15989/files#r1045904676

Reply:
removed here https://github.com/brave/brave-core/pull/15989/commits/80862768735dfc5ce4546e5d0608b31d51b044ef